src/types.jl

```diff
@@ -130,3 +130,13 @@ Returns hyperedges for a given vertex
 * `v_id` : an identifier of a vertex
 """
 @inline gethyperedges(h::Hypergraph, v_id::Int) = h.v2he[v_id]
+
+function add_vertex!(h::Hypergraph{T};hyperedges::Dict{Int,T} = Dict{Int,T}()) where T <: Real
+    push!(h.v2he,hyperedges)
```

Comment:
this method and also one below is wrong as it updates only one collection while both collections vertices->hyperedges and hyperedges->vertices should be updated in both cases.